### ISSUE TITLE
Issues with flowplayer

### ISSUE THREAD

**Issue by pmallek** [google/material-design-lite#1595]
Hello, i have two issues while working with mdl + flowplayer (quiet popular video player: http://flowplayer.org). First i asked about that on flowplayer forum but devs said that problem is related to "my own css" = mdl css. Here i made two sample pages to not make things too complicated.

Sample 1: http://flowplayer.zobaczzwiastun.pl/sampleMdl.php
Sample 2: http://flowplayer.zobaczzwiastun.pl/sampleMdlAdsense.php

So first problem appears only on my ipad (dont have any other Apple device to test it, not sure if its just mobile iOS problem or overall iOS) when i fullscreen video, player goes behind all mdl items. Check Sample 1, header is on top of player.

Second issue is related to adsense for video plugin: https://flowplayer.org/docs/asf.html. Sample 2 shows flowplayer + mdl + adsense for video plugin. When you play video after 3 second show up image_text ad, since this moment i can no longer pause video by clicking on it, only buttons from controlbar works (there is little space above controlbar where its possible to click and pause/resume video but its just impossible for regular user).

**Comment by surma** [comment]
Hey @pmallek. I am not familiar with FlowPlayer but this is definitely something you should take to StackOverflow using the `material-design-lite` tag. This falls under support and is not a bug with MDL at its core.

My guess would be that the `z-index` that we set in MDL are somehow incompatible with iOS fullscreen and AdSense, but I am not sure (don’t have an iOS device at hand right now).

**Comment by pmallek** [comment]
@surma we dont have to talk about fullscreen problem on iOS, i have some solution for this issue. However adsense plugin is pretty big problem and i cant solve it. Believe me if it would be z-index or any other simple css thing i would never post it in any public place:) I did some more research, if i dont import mdl css file, problem still exists. I also commented every component register in material.js file. After that player works fine. I checked all components one by one to see which one cause problem and its layout. How ad is working in player? In my sample it loads iframe (after 3 seconds) and here something bad happen with this iframe (if i manually, for example in chrome delete iframe with ad from dom, i can again resume/pause player). So question is, is there any way how mdl js can interact with iframe?

Summary: Something bad happen with ad iframe when layout component is on and this make troubles with player usage.

I also asked about that on stack but community isnt to big yet. http://stackoverflow.com/questions/32560839/flowplayer-and-material-design-lite
